app/views/suggestions/favorites.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Your Favorites</h1>
+
+<% if @suggestion %>
+<%= @suggestion.each do |pair| %>
```

Comment:
This spacing, though! hehe. Are you using Sublime? Each indentation should be two spaces.